commit 333cb77bbe10d449d968f36285b6a5d067cbac2d
Author: Yang Bo <yangbo@linuxdeepin.com>
Date:   Sun Jan 3 18:41:45 2016 +0800

    UBUNTU: SAUCE: (no-up) Input: synaptics - add second variant of two-button clickpad
    
    This is necessary for clickpad detection of Synaptics trackpads in Dell
    Mini 10 series of laptops.
    
    no-up comments: Keep both, so long as they continue to apply cleanly.
    The patches apply only to a couple of old Dell minis, and Dell has said
    they don't intend to use those touchpads again. Upstreaming these
    patches stalled due to lack of information/response, and continuing to
    pursue it probably isn't worth the effort, so they can be marked no-up.
    There's no harm in keeping the patches, but if they become a problem
    they can be dropped.
    
    Signed-off-by: Chase Douglas <chase.douglas@canonical.com>
    Acked-by: Seth Forshee <seth.forshee@canonical.com>
    Acked-by: Andy Whitcroft <andy.whitcroft@canonical.com>
    Signed-off-by: Tim Gardner <tim.gardner@canonical.com>
    Signed-off-by: Yang Bo <yangbo@linuxdeepin.com>

diff --git a/drivers/input/mouse/synaptics.h b/drivers/input/mouse/synaptics.h
index 56faa7e..c20f32c 100644
--- a/drivers/input/mouse/synaptics.h
+++ b/drivers/input/mouse/synaptics.h
@@ -85,6 +85,7 @@
  */
 #define SYN_CAP_CLICKPAD(ex0c)		((ex0c) & 0x100000) /* 1-button ClickPad */
 #define SYN_CAP_CLICKPAD2BTN(ex0c)	((ex0c) & 0x000100) /* 2-button ClickPad */
+#define SYN_CAP_CLICKPAD2BTN2(ex0c)	((ex0c) & 0x200000) /* 2-button ClickPad */
 #define SYN_CAP_MAX_DIMENSIONS(ex0c)	((ex0c) & 0x020000)
 #define SYN_CAP_MIN_DIMENSIONS(ex0c)	((ex0c) & 0x002000)
 #define SYN_CAP_ADV_GESTURE(ex0c)	((ex0c) & 0x080000)
